packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/SummarySheet.tsx

```diff
@@ -84,11 +87,57 @@ const SummarySheet = (props: Props) => {
     `,
     meetingRef
   )
+  const summaryRef = useRef<HTMLTableElement | null>(null)
   const {id: meetingId, meetingType, taskCount} = meeting
   const isDemo = !!props.isDemo
 
+  const downloadPDF = async () => {
+    const summaryTable = summaryRef.current
+    if (!summaryTable) return
+
+    const canvas = await html2canvas(summaryTable, {
+      scrollY: -window.scrollY,
+      useCORS: true
+    })
+    const pdf = new jsPDF()
+    const imgData = canvas.toDataURL('image/png')
+
+    const pageWidth = 210 // A4 width in mm
+    const pageHeight = 297 // A4 height in mm
+    const imgWidth = canvas.width * 0.264583 // convert pixel to mm
+    const imgHeight = canvas.height * 0.264583 // convert pixel to mm
+
+    const scale = pageWidth / imgWidth
+    const scaledWidth = imgWidth * scale
+    const scaledHeight = imgHeight * scale
+
+    let remainingHeight = scaledHeight
+    let isFirstPage = true
+
+    while (remainingHeight >= 0) {
+      const position = remainingHeight - scaledHeight
+      if (!isFirstPage) {
+        pdf.addPage()
+      } else {
+        isFirstPage = false
+      }
+
+      pdf.addImage(imgData, 'PNG', 0, position, scaledWidth, scaledHeight)
```

Comment:
-2 By creating the pdf this way, we end up with a series of screenshots just added to the pdf. The MacOS preview is misleading as it does some OCR, but checking with another PDF viewer you can see that we did not render text. Despite missing search functionality etc. it also renders unsharp text.

The better way would be to use [`pdf.html`](https://rawgit.com/MrRio/jsPDF/master/docs/module-html.html#~html) function which does not scale correctly for me out of the box, but renders text as such.

Reply:
I've updated the PR (and the Loom in the description) to use `puppeteer`, which creates the PDF on the server. The text should now be searchable 👍

Reply:
@nickoferrall without diving too deep into it right now, but did you check the system requirements for puppeteer? I have some concerns about chromium impacting our GQLExecutors when running on the same node. That might require some more work on our infrastructure first. What are your thoughts?

Reply:
I'd like to avoid adding puppeteer to our runtime if at all possible, it's BIG with a lot of dependencies & something about trusting a blob of HTML from a client feels like it could open us up to a 0-day. (e.g. does puppeteer sanitize all SVGs? <script> tags? clickHandlers?). It's generally used on trusted elements & I just haven't seen enough about it on untrusted HTML structures.

Before we have to go down that road, I'm curious, and this is just because I don't have all the context here, why not just open the print page dialog? Every windows, mac, linux computer comes with a "Print to PDF" printer installed. It will be faster because there's no server involved, more customizable because they can set the name, and best of all-- keep our codebase simpler!

<img width="379" alt="Screenshot 2023-09-18 at 11 15 33 AM" src="https://github.com/ParabolInc/parabol/assets/5514175/10ef96b0-a41b-4cd7-a66c-38f7495da991">

If we absolutely no question about it MUST generate a PDF on our server, we should generate the HTML on the server & then ideally use a smaller package to do the print to pdf (pdfjs?)

Reply:
Love this idea! I've created this PR https://github.com/ParabolInc/parabol/pull/8845 with the print page dialog and will close this one 👍